905.py

```diff
@@ -0,0 +1,15 @@
+class Solution:
+    def sortArrayByParity(self, A: List[int]) -> List[int]:
+        i, j = 0, len(A) - 1
+
+        while i < j:
+            if A[i] % 2 > A[j] % 2:
```

Comment:
I can tell that this line is trying to determine whether `A[i]` is odd and `A[j]` is even because `1 > 0`. However, this meaning is not explicitly conveyed through the code itself--it requires an extra step for the reader to figure it out.